Source/ViewControllers/BrickCollectionView.swift

```diff
@@ -501,7 +509,10 @@ extension BrickCollectionView: AsynchronousResizableDelegate {
         weak var weakLayout = self.layout
         self.performBatchUpdates({
             weakLayout?.updateHeight(indexPath, newHeight: newHeight)
-            }, completion: completion)
+        }, completion: { completed in
+            let _ = self
```

Comment:
Could this be the one that is the problem? All the other ones capture `self` in the `performBatchUpdates`-block